README.md

```diff
@@ -153,6 +154,7 @@ particlesJS.load('particles-js', 'assets/particles.json', function() {
 
 key | option type / notes | example
 ----|---------|------
+`.fps_limit` | Limits the fps of the animation redraw, set to 0 for no limit | `30`
```

Comment:
```suggestion
`.fps_limit` | Limits the fps of the animation redraw, set to 0 for no limit | `60`
```
Should this be 60?

Reply:
It should be merged is what it should be!